packages/story-utils/src/getStories.ts

```diff
@@ -0,0 +1,33 @@
+import path from 'path';
+import type { Parameters, Story } from '@storybook/react/types-6-0';
+import glob from 'glob';
+import merge from 'lodash/merge';
+
+type StoryData = {
+  componentTitle: string;
+  name: string;
+  parameters: Parameters;
+  storyFn: Story<unknown>;
+};
+
+export default function getStories(globPath = '**/*.stories.*'): StoryData[] {
```

Comment:
This is essentially the implementation before https://github.com/chanzuckerberg/along/pull/704.

@dcwither, once this is in feel free to update as necessary to implement auto-snapshot testing.